requirements-test.txt

```diff
@@ -50,7 +50,7 @@ repoze.lru==0.7
 requests==2.21.0
 six==1.12.0
 SQLAlchemy==1.3.0
-thrift==0.12.1
+thrift>=0.12.1
```

Comment:
🔕 I'm a little nervous about dependencies that can upgrade themselves. I generally prefer pinning to the exact version.

Reply:
Yeah, it's a bit unclear what versions of thrift we support at the moment.  Mostly because they seem to be different on mac and linux :|